modules/schematics/src/feature/__snapshots__/index.spec.ts.snap

```diff
@@ -1,5 +1,177 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Feature Schematic should create all files of a feature with an entity 1`] = `
+"import { createActionGroup, emptyProps, props } from '@ngrx/store';
+import { Update } from '@ngrx/entity';
+
+import { Foo } from './foo.model';
+
+export const FooActions = createActionGroup({
+  source: 'Foo/API',
+  events: {
+    'Load Foos': props<{ foos: Foo[] }>(),
+    'Add Foo': props<{ foo: Foo }>(),
+    'Upsert Foo': props<{ foo: Foo }>(),
+    'Add Foos': props<{ foo: Foo[] }>(),
+    'Upsert Foos': props<{ foo: Foo[] }>(),
+    'Update Foo': props<{ foo: Update<Foo> }>(),
+    'Update Foos': props<{ foos: Update<Foo>[] }>(),
+    'Delete Foo': props<{ id: string }>(),
+    'Delete Foos': props<{ ids: string[] }>(),
+    'Clear Foos': emptyProps(),
+  }
+});
+"
+`;
+
+exports[`Feature Schematic should create all files of a feature with an entity 2`] = `
+"import { createFeature, createReducer, on } from '@ngrx/store';
+import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
+import { Foo } from './foo.model';
+import { FooActions } from './foo.actions';
+
+export const foosFeatureKey = 'foos';
+
+export interface State extends EntityState<Foo> {
+  // additional entities state properties
+}
+
+export const adapter: EntityAdapter<Foo> = createEntityAdapter<Foo>();
+
+export const initialState: State = adapter.getInitialState({
+  // additional entity state properties
+});
+
+export const reducer = createReducer(
+  initialState,
+  on(FooActions.addFoo,
+    (state, action) => adapter.addOne(action.foo, state)
+  ),
+  on(FooActions.upsertFoo,
+    (state, action) => adapter.upsertOne(action.foo, state)
+  ),
+  on(FooActions.addFoos,
+    (state, action) => adapter.addMany(action.foos, state)
+  ),
+  on(FooActions.upsertFoos,
+    (state, action) => adapter.upsertMany(action.foos, state)
+  ),
+  on(FooActions.updateFoo,
+    (state, action) => adapter.updateOne(action.foo, state)
+  ),
+  on(FooActions.updateFoos,
+    (state, action) => adapter.updateMany(action.foos, state)
+  ),
+  on(FooActions.deleteFoo,
+    (state, action) => adapter.removeOne(action.id, state)
+  ),
+  on(FooActions.deleteFoos,
+    (state, action) => adapter.removeMany(action.ids, state)
+  ),
+  on(FooActions.loadFoos,
+    (state, action) => adapter.setAll(action.foos, state)
+  ),
+  on(FooActions.clearFoos,
+    state => adapter.removeAll(state)
+  ),
+);
+
+export const foosFeature = createFeature({
+  name: foosFeatureKey,
+  reducer,
+  extraSelectors: ({ selectFoosState }) => ({
+    ...adapter.getSelectors(selectFoosState)
+  }),
+});
+
+export const {
+  selectIds,
+  selectEntities,
+  selectAll,
+  selectTotal,
+} = foosFeature;
```

Comment:
> @timdeschryver I've removed the Actions spec file as well as the Selector/Selector spec files from the code in your commit. The Entity schematic doesn't have templates for any of these files at the moment.
> 
> You [mentioned](https://github.com/ngrx/platform/pull/3850#discussion_r1172919625) that we should let the Entity schematic handle these things for now. If you/the team would prefer that selectors be generated as well, I can update the Feature schematic to run the Selector schematic every time.
> 
> Snapshots have been updated!

Thanks! The selectors are already included within this feature file.
If I'm missing something feel free to correct me.